db/seeds/jokes.rb

```diff
@@ -33,3 +33,12 @@
 puts Joke.create(lead: "I wouldn't buy anything with velcro.", punchline: "It's all a rip-off!")
 puts Joke.create(lead: "Did you hear about the restaurant on the moon?", punchline: "Great food, but no atmosphere!")
 puts Joke.create(lead: "When you grow up, you'll thank all the sidewalks...", punchline: "for keeping you off the streets!")
+puts Joke.create(lead: "Whats more amazing than a talking dog", punchline: "A Spelling Bee.")
+puts Joke.create(lead: "Why did the snowman call his dog frost", punchline: "Because frost bites.")
```

Comment:
Line is too long. [99/80]<br>Do not write to stdout. Use Rails' logger if you want to log.